builds/fluent_bit_local.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x -e
+DESTDIR=$1
+mkdir -p $DESTDIR
+
+cd submodules/fluent-bit
+mkdir -p build
+cd build
+# CMAKE_INSTALL_PREFIX here will cause the binary to be put at
```

Comment:
Comment seems no longer relevant

Reply:
Removed in [cfcc68f](https://github.com/GoogleCloudPlatform/ops-agent/pull/1722/commits/cfcc68fa1f5cb39c0f7d3c1749bd05a166fb5c18)